src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -102,7 +104,9 @@ public CollectionResource(MeteredCommandProcessor meteredCommandProcessor) {
                         InsertOneCommand.class,
                         InsertManyCommand.class,
                         UpdateManyCommand.class,
-                        UpdateOneCommand.class
+                        UpdateOneCommand.class,
+                        AddIndexCommand.class,
```

Comment:
Same as earlier, should probably be in alphabetic order.

Reply:
I added these at the end because this is specific to tables. All other commands are common to both collections and tables.